webpack.config.js

```diff
@@ -102,8 +102,7 @@ module.exports = (env, args = {}) => {
         },
         {
           test: /\.ya?ml$/,
-          // Delete yaml-flat-loader and use a regular yaml loader when migration to i18next is complete
-          use: [{ loader: "json-loader" }, { loader: "yaml-flat-loader" }],
+          loader: "yaml-loader",
```

Comment:
Is it okay you deleted `json-loader` here along with `yaml-flat-loader`? The comment said to only remove the yaml flat loader and replace it with yaml-loader.